static/src/javascripts/lib/ajax.js

```diff
@@ -13,7 +12,25 @@ const ajax = (params) => {
         options.crossOrigin = true;
     }
 
-    const r = reqwest(options);
+    const { url } = options;
+    const headers = new Headers();
+    if(options.headers !== undefined) headers.append( options.headers );
+    if(options.contentType !== undefined)
+        headers.append('Content-Type', options.contentType );
+    let headerSize = 0;
+    headers.forEach(() => { headerSize++ });
+
+
+    const initArray = Object.entries({
```

Comment:
Is the an advantage for using filter not null with `entries` and `fromEntries` over just setting the keys to undefined?